hooks/settings.py

```diff
@@ -0,0 +1,187 @@
+                  <param name="Adaptive B-frame decision" internalname="adaptive_b_frames" description="" type="bool" value="1" />
+                  <param name="Automatic scene detection" internalname="vcsd_mode" description="" type="bool" value="1" />
+                  <param name="Rate control" internalname="ratecontrol" description="" type="section" />
+                  <param name="Rate control method" internalname="bit_rate_mode" description="" type="enum" value="Average bitrate" />
+                  <param name="Bitrate" internalname="bit_rate" description="Bitrate in kbit/s. (Used in bitrate mode.)" type="int" min="1" max="1000000" value="8000" />
+                  <param name="Max bitrate" internalname="max_bit_rate" description="Max bitrate in kbit/s. Must be greater than the bitrate. (Used in bitrate mode.)" type="int" min="1" max="1000000" value="9999" />
+                  <param name="I-frame quantizer" internalname="quant_pI" description="Quantization value for I-frames. (Used in quantizer mode.)" type="int" min="1" max="51" value="24" />
+                  <param name="P-frame quantizer" internalname="quant_pP" description="Quantization value for P-frames. (Used in quantizer mode.)" type="int" min="1" max="51" value="25" />
+                  <param name="B-frame quantizer" internalname="quant_pB" description="Quantization value for B-frames. (Used in quantizer mode.)" type="int" min="1" max="51" value="27" />
+                  <param name="Minimum quantizer" internalname="min_quant" description="Minimum quantization value. (Used in both mode.)" type="int" min="1" max="51" value="0" />
+                  <param name="Maximum quantizer" internalname="max_quant" description="Maximum quantization value. (Used in both mode.)" type="int" min="1" max="51" value="51" />
+                  <param name="Advanced" internalname="advanced" description="" type="section" />
+                  <param name="Optimize rate-distortion cost" internalname="rd_optimization" description="" type="bool" value="1" />
+               </params>
+            </codec>
+         </provider>
+      </essence>
+   </format>
+</codecprofile>
+        """
+        
+        path = self._write_content_to_file(xml, "quicktime_settings.cdxprof")
+        return path
+    
+    def _generate_profile_xml(self):
+        """
+        Generate flame export profile settings suitable for generating a single quicktime 
+        file to represent an entire sequence.
+        
+        :returns: path to export preset xml file
+        """
+        
+        # this profile is a 8-bit QuickTime file (H.264 1280x720 8Mbits) with 
+        # preset High_Blu_Ray_8Mbits, suitable for upload to Shotgun Review.
+        #
+        # for more details around quicktime settings and shotgun, see
+        # https://support.shotgunsoftware.com/entries/26303513-Transcoding
+        
+        xml = """<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Feel like it would be better to have these constants defined in a xml_templates.py file and imported, making the core more readable. Or may be just defined at the top of the file, and not inline in the method.